.github/workflows/.windows.yml

```diff
@@ -231,7 +231,7 @@ jobs:
           # Also prepend ./... to the matrix. This is a special case to run "Test integration" step exclusively.
           matrix="$(gotestlist -d ${{ env.ITG_CLI_MATRIX_SIZE }} ./...)"
           matrix="$(echo "$matrix" | jq -c '. |= ["./..."] + .')"
-          echo "::set-output name=matrix::$matrix"
+          echo "matrix=$matrix" >> $GITHUB_STATE
```

Comment:
Looks it breaks the matrix integration tests on windows. Looks empty now: https://github.com/moby/moby/actions/runs/3439139750/jobs/5736096920

Reply:
Ah it should be:

```suggestion
          echo "matrix=$matrix" >> $GITHUB_OUTPUT
```